docs/_sass/_fonts.scss

```diff
@@ -1,92 +1,15 @@
 @font-face {
-  font-family: GTAmericaExp-Regular;
-  font-weight: 100;
-  font-style: normal;
-  src: url("https://www.expensify.com/font/GT-America-Standard-Light.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Light.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Light.woff2") format("woff2");
-}
-
-@font-face {
-  font-family: GTAmericaExp-Regular;
-  font-weight: 200;
-  font-style: normal;
-  src: url("https://www.expensify.com/font/GT-America-Standard-Thin.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Thin.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Thin.woff2") format("woff2");
-}
-
-@font-face {
-  font-family: GTAmericaExp-Regular;
+  font-family: ExpensifyNeue-Regular;
```

Comment:
Let's make all of these font families just say `ExpensifyNeue` because `Regular` is actually just a weight/style of the family.